go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/mt-sre/devkube
+
+go 1.17
+
+require (
+	github.com/go-logr/logr v1.2.0
+	github.com/magefile/mage v1.12.1
+	github.com/openshift/addon-operator/apis v0.0.0-20220111092509-93ca25c9359f
+	github.com/stretchr/testify v1.7.0
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.23.0
+	k8s.io/apiextensions-apiserver v0.23.0
+	k8s.io/apimachinery v0.23.1
+	k8s.io/client-go v0.23.0
+	sigs.k8s.io/controller-runtime v0.11.0
+	sigs.k8s.io/yaml v1.3.0
+)
+
+require (
```

Comment:
Im curious why do we have 2 require sections? I am not familiar with this.

Reply:
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.
https://go.dev/doc/go1.17 :tada: